optuna/testing/visualization.py

```diff
@@ -16,12 +14,6 @@ def prepare_study_with_trials(
     """Prepare a study for tests.
 
     Args:
-        less_than_two: If :obj:`True`, create a study with two/four hyperparameters where
-            'param_a' (and 'param_c') appear(s) only once while 'param_b' (and 'param_d')
-            appear(s) twice in `study.trials`.
-        more_than_three: If :obj:`True`, create a study with two/four hyperparameters where
-            'param_a' (and 'param_c') appear(s) only three times while 'param_b' (and 'param_d')
-            appear(s) four times in `study.trials`.
```

Comment:
All usages of this option are removed at https://github.com/optuna/optuna/pull/3701